### DISCUSSION TITLE
Windows / Game Pass Version

### DISCUSSION THREAD

**Discussion by Alex444Alex** [zencq/NomNom#67] [Q&A] [answered]
Is there a tutorial or something I can read that will explain how to set it up on Windows / Game Pass Version? please an ty

**Comment by nextdayy** [comment]
you literally just download it and open it and it works. simple.

**Comment by cengelha** [comment, marked as answer]
Sorry @Alex444Alex, I totally forgot your question... As nxtdaydelivery said. If have played at least once, so you have a save on your disk, NomNom will automatically detect and load them. Then just use it ;)
